pyartcd/pyartcd/pipelines/scan_fips.py

```diff
@@ -13,42 +13,45 @@
 
 
 class ScanFips:
-    def __init__(self, runtime: Runtime, version: str, nvrs: Optional[list]):
+    def __init__(self, runtime: Runtime, nvrs: Optional[list]):
         self.runtime = runtime
-        self.version = version
         self.nvrs = nvrs
+        self.active_ocp_versions = [4.12, 4.13, 4.14, 4.15, 4.16, 4.17]
 
         # Setup slack client
         self.slack_client = self.runtime.new_slack_client()
         self.slack_client.bind_channel("#art-release")
 
     async def run(self):
-        cmd = [
-            "doozer",
-            "--group",
-            f"openshift-{self.version}",
-            "--data-path",
-            "https://github.com/openshift-eng/ocp-build-data",
-            "images:scan-fips",
-            "--nvrs",
-            f"{','.join(self.nvrs)}"
-        ]
+        results = {}
 
-        _, result, _ = await exectools.cmd_gather_async(cmd, stderr=True)
+        for version in self.active_ocp_versions:
+            cmd = [
+                "doozer",
+                "--group",
+                f"openshift-{version}",
+                "--data-path",
+                "https://github.com/openshift-eng/ocp-build-data",
```

Comment:
This shouldn't be hardcoded here. We either use the default value, or let the user to override through the job parameter.

Reply:
well we don't really need a param for this, but I guess I can set a default in the pipeline run / pipeline task